### ISSUE TITLE
Wio Terminal missing support for wifi, bluetooth and USB in ArduPy

### ISSUE THREAD

**Issue by ruppde** [Seeed-Studio/ArduPy#18]
**Describe the bug**
Opposed to seeeds website, the python support for wifi, bluetooth, accelerometer, IR emitter, light sensor and USB is missing. And it looks like it will never come since the last update to ardupy was in August 2020.

**To Reproduce**
1. Look at the webpage: https://www.seeedstudio.com/Wio-Terminal-p-4509.html
"Compatible with Arduino and MicroPython, Wio Terminal is an ATSAMD51-based microcontroller with wireless connectivity powered by Realtek RTL8720DN"
2. Look at Ardupy, nothing there.

**Expected behavior**
Support for wifi, bluetooth, accelerometer, IR emitter, light sensor and USB as promised on webpage.

**Comment by lakshanthad** [comment]
Hi,
The webpage and the wiki pages are already updated and highlighted that Wio Terminal wireless connectivity is only supported by Arduino at the moment. Thank you for pointing it out.

We initially pointed out that ArduPy is also supported hoping to bring more functionalities later. Please understand that ArduPy is quite new and we plan to gradually bring more functions in the future.

- webpage: https://www.seeedstudio.com/Wio-Terminal-p-4509.html
- wiki: https://wiki.seeedstudio.com/Wio-Terminal-Getting-Started/

Also, I have replied to the other functions issues here:
https://github.com/lakshanthad/Wio_Terminal_Classroom_Ardupy/issues/3

Best Regards,
Lakshantha

**Comment by ruppde** [comment]
ok, thx

**Comment by ruppde** [comment]
@lakshanthad this issue isn't solved, please keep it open.

**Comment by lakshanthad** [comment]
Hi @2d4d,

We have released the latest ardupy-aip firmware to fix the existing bugs. 

https://github.com/Seeed-Studio/ardupy-aip

Please uninstall your existing installation by:
```sh
pip3 uninstall ardupy-aip
```

And, install the latest version by:
```sh
pip3 install ardupy-aip
```

After installing, please run the following to make sure you have installed v0.9.1 which is the latest version:
```sh
aip -V
```

Best Regards,
Lakshantha

**Comment by ruppde** [comment]
hello @lakshanthad,

ok, great, where can I get the new firmware? (as far as I understand it, ardupy-aip is just the tool for the PC?) I didn't find a new firmware on https://wiki.seeedstudio.com/ArduPy/ 

regards
arnim

**Comment by lakshanthad** [comment]
Yes. You are right. Sorry. ardupy-aip is just a tool. You can use it to build your own ArduPy firmware and flash into Wio Terminal. Previously there were some issues when running this tool. So they are fixed now.

Use `aip help` to view the available commands

Follow this :
https://wiki.seeedstudio.com/ArduPy

Best Regards,
Lakshantha

**Comment by ruppde** [comment]
@lakshanthad hi, any news on the wifi + bluetooth support for wio terminal? would be very useful to properly use it ;)

**Comment by Techy52** [comment]
@lakshanthad any news on including  wifi + bluetooth support for wio terminal?

**Comment by kelchm** [comment]
It would be great to hear an update on this -- not being able to use wifi and bluetooth with ArduPy on the Wio Terminal has been a big disappointment.

**Comment by genwch** [comment]
so disappointed for missing wifi support with circuitpy or ardupy

**Comment by ruppde** [comment]
> so disappointed for missing wifi support with circuitpy or ardupy

yep, I doubt they'll ever change it :( go for m5stack, that's fully supported with python.
